www/src/components/shell/onboarding/cloud/CloudCredentials.js

```diff
@@ -42,44 +46,51 @@ function CloudCredentials() {
   return (
     <>
       <OnboardingCard title="Configure cloud credentials">
-        <FormField
-          width="100%"
-          marginTop="large"
-          marginBottom="large"
-          label="Cloud provider"
-        >
-          <Select
+        {/* Div wrapper needed for bottom padding show up
+        when content overflows */}
+        <div>
+          <FormField
             width="100%"
-            onChange={({ target: { value } }) => {
-              setProvider(value)
-            }}
-            value={provider}
-          >
-            {CLOUDS.map(cloud => (
-              <MenuItem
-                key={cloud}
-                value={cloud}
-              >
-                {cloud}
-              </MenuItem>
-            ))}
-          </Select>
-        </FormField>
-        <Box>
-          {createElement(form, {
-            workspace, setWorkspace, credentials, setCredentials,
-          })}
-        </Box>
-        {open && (
-          <Drop
-            target={ref.current}
-            onClickOutside={close}
-            onEsc={close}
+            marginTop="large"
+            marginBottom="large"
+            label="Cloud provider"
           >
-            <Box width="250px" />
-          </Drop>
-        )}
-        {exceptions && <Exceptions exceptions={exceptions} />}
+            <Select
+              width="100%"
+              onChange={({ target: { value } }) => {
+                setProvider(value)
+              }}
+              value={provider}
+            >
+              {CLOUDS.map(cloud => (
```

Comment:
Could we add a growthbook feature flag to control if azure is included in this list?

Reply:
Actually wait I'll do it since I have growthbook access